doc/classes/GridMap.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GridMap" inherits="Spatial" category="Cpre" version="3.0.alpha.custom_build">
+	<brief_description>
+		Node for 3D tile-based maps.
+	</brief_description>
+	<description>
+		Node for 3D tile-based maps.
```

Comment:
Don't duplicate the brief description, try to develop it instead. E.g. (note, I don't know how the node works, it's just an imaginary example)
`[code]GridMap[/code] lets you place meshes on a grid interactively. It works both from the editor and helps you create in-game level editors.`